lib/cryptoexchange/exchanges/bitholic/services/pairs.rb

```diff
@@ -0,0 +1,25 @@
+module Cryptoexchange::Exchanges
+  module Bitholic
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+
+        def fetch
+          output = super
```

Comment:
Should read the pairs from https://api.bitholic.com/public/ticker as well instead of using `yml` user defined